app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -30,6 +31,7 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
     private val readerViewFeature = ViewBoundFeatureWrapper<ReaderViewIntegration>()
     private val webExtToolbarFeature = ViewBoundFeatureWrapper<WebExtensionToolbarFeature>()
 
+    @ExperimentalCoroutinesApi
```

Comment:
I think we'd be fine removing all these annotations with compiler arg (similar to [Focus](https://github.com/mozilla-mobile/focus-android/blob/master/app/build.gradle#L118)).